Automattic/wp-calypso
# Add/sites thumbnails

#### Proposed Changes

* Add site thumbnail component in sites dashboard

#### Testing Instructions

- Go to `/sites-dashboard`
- Verify sites thumbnail shows mshot image for launched sites
- Verify sites thumbnail shows site icon for private/coming soon sites when icon is present
- Verify sites thumbnail shows first letter of site domain when site icon is not present
- Verify background color is selected from site icon for private/coming soon sites



***Limitation**
Currently the hook that gets the primary color from the site icon encounters CORS issues when fetching image from atomic and some other sites (see console logs). Presently those sites retain a default background and colors are only pick from sites with access to the site icon.

#### Pre-merge Checklist






**Note**: background color now is a randomly generated color, that will be fixed in another PR

Related to https://github.com/Automattic/wp-calypso/issues/65171 and https://github.com/Automattic/wp-calypso/issues/65942
